lms/templates/login.html

```diff
@@ -130,6 +130,8 @@
   </script>
 </%block>
```

Comment:
@andy-armstrong Do you think it matters where in the HTML page this include occurs? Before I think it would have been towards the top of the page, as it was included by main.html (via navigation). I put it near the top here, but it is after the pagetitle and bodyclass blocks.